src/main/java/codeu/service/ChatService.java

```diff
@@ -13,26 +13,36 @@
 
```

Comment:
This service is used by the ChatBot to send messages to users; thats why the notificationService is called from here